cli/sawtooth_cli/identity.py

```diff
@@ -13,14 +13,16 @@
 # limitations under the License.
 # ------------------------------------------------------------------------------
 
+from base64 import b64decode
```

Comment:
These changes should be squashed into the "Add identity to sawtooth cli." commit